docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -550,6 +552,22 @@ iceberg.rest-catalog.security=OAUTH2
 iceberg.rest-catalog.oauth2.token=***
 ```
 
+`iceberg.rest-catalog.security` must be `GOOGLE` when connecting to BigLake metastore
```

Comment:
Do we have the place to see each metastore's functionality details, i,e support "rename" or not